force-app/main/default/classes/tasks/BasicTasks.cls

```diff
@@ -3,7 +3,7 @@ public class BasicTasks {
   public static Long sumIntegers(Integer a, Integer b) {
     // ignore public static.. Focus on variables
     //write your code below this line
-    Integer sum = 0;
+    Long sum = a + b;
```

Comment:
I just double-checked this task. There's an issue to be fixed. Try running in the terminal to see the test results. Everything else looks good

```
sfdx force:apex:test:run -t CollectionsTasksTest,BasicTasksTest -w 10
```